.github/workflows/markdown.yaml

```diff
@@ -2,24 +2,24 @@ name: "Docs: Markdown lint"
 
 on: [push, pull_request]
 
+permissions:
+  contents: read
+
 jobs:
   markdown-lint:
     runs-on: ubuntu-22.04
     steps:
-    - uses: actions/checkout@v4
-      with:
-        fetch-depth: 0
-    - uses: tj-actions/changed-files@v44
-      id: changed-files
-      with:
-        files: |
-          docs/src/snap/**
-          docs/src/charm/**
-    - uses: DavidAnson/markdownlint-cli2-action@v16
-      if: steps.changed-files.outputs.any_changed == 'true'
-      with:
-        config: 'docs/src/.markdownlint.json'
-        globs: ${{ steps.changed-files.outputs.all_changed_files }}
-        separator: ","
-
-
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: tj-actions/changed-files@v44
+        id: changed-markdown-files
+        with:
+          files: |
+            docs/**/*.md
```

Comment:
can we add a 
!docs/src/_parts/**
or whatever. Some of those files will fail the rules because they are bits of md rather than complete files.